iOS_SDK/OneSignalSDK/UnitTests/UnitTestCommonMethods.m

```diff
@@ -215,14 +215,37 @@ + (void)initOneSignalAndThreadWait {
 
 + (void)foregroundApp {
     UIApplicationOverrider.currentUIApplicationState = UIApplicationStateActive;
-    UIApplication *sharedApp = [UIApplication sharedApplication];
-    [sharedApp.delegate applicationDidBecomeActive:sharedApp];
+    
+    if (@available(iOS 13.0, *)) {
```

Comment:
I see we have this iOS 13 and sceneManifest check in at least three spots. Could you make this a single method we could call? Such as `isAppUsingUIScene`?